interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -101,100 +50,37 @@ namespace cling {
   public:
     IfStmtInjector(Sema& S) : m_Sema(S), m_Context(S.getASTContext()),
     m_LookupResult(0) {}
-    CompoundStmt* Inject(CompoundStmt* CS) {
-      NodeContext result = VisitCompoundStmt(CS);
-      return cast<CompoundStmt>(result.getStmt());
+    void Inject(CompoundStmt* CS) {
```

Comment:
Can we just call `Visit(CS)` from the `NullDerefTransformer` and get rid of `Inject()`?